### ISSUE TITLE
REPL :complete section unnecessary

### ISSUE THREAD

**Issue by milesfrain** [JordanMartinez/purescript-jordans-reference#401]
## Context

```
Section:
  Getting Started

Issue Type:
  Bug

Folder/File(s):
  - Getting Started/The REPL.md
```

https://github.com/JordanMartinez/purescript-jordans-reference/blob/development/00-Getting-Started/03-The-REPL.md#complete

## Body

The REPL supports tab completion now, so this command and section are probably not necessary anymore.

If this section remains, `:c a` should be changed to `:co a` to disambiguate from `:clear`.

**Comment by JordanMartinez** [comment]
Thanks for finding this. We can make that change in the mean time, but the real issue is that the REPL should probably no longer have a `:complete` command in the first place. (I'll open up an issue on that repo)

Do you know the PS version in which the tab completion was first supported?

**Comment by milesfrain** [comment]
> Do you know the PS version in which the tab completion was first supported?

Nope. My only PS experience is on the latest versions.

**Comment by hdgarrood** [comment]
The PureScript repl has had tab completion for at least 4 years. @JordanMartinez Do you remember what gave you the impression that it wasn't supported?

**Comment by JordanMartinez** [comment]
@hdgarrood Unfortunately, I don't. It might have been that command's description that made me think that. Either way, thanks for correcting me.

@milesfrain Thanks for bringing this up again. Looks like the clearer explanation for that command is that it's not meant to be used by humans and should otherwise be ignored as tab completion works fine.

**Comment by JordanMartinez** [comment]
Fixed in 22679e4594d1acf4fa18765dd1c99bb568b7deb3

This will be included in next release.
